Source/Urho3D/UI/ToolTip.cpp

```diff
@@ -74,10 +74,15 @@ void ToolTip::Update(float timeStep)
             SharedPtr<UIElement> target = it->Lock();
             if (!target)
                 it = altTargets_.Erase(it);
-            else if (hovering = target->IsHovering() && target->IsVisibleEffective())
-                break;
             else
-                ++it;
+            {
+                hovering = target->IsHovering()
+
+                if (hovering && target->IsVisibleEffective())
```

Comment:
Bug injected here.

Before: `hovering = target->IsHovering() && target->IsVisibleEffective()`
After: `hovering = target->IsHovering()`

Reply:
it won't compile without `;`

Reply:
whoops

Reply:
See, this is exactly the reason why I think old code is evil and has to be changed.

Reply:
Yeah I definitely got distracted by the syntax...